### PR TITLE
Load and save all ini values the same way

### LINKED ISSUE #1238: [controls] options should be in the config by default
This improves their discoverability and makes it easier for users to change

### PR DESCRIPTION
**Pull request by AJenbo**
This exposes all options clearly in the ini file and makes things a bit
more manageable

Fixes #1238

### CODE REVIEW

**Review by julealgon — APPROVED** [review]
Very nice cleanup.

File: Source/diablo.cpp [hunk 1]
```diff
@@ -390,11 +390,12 @@ BOOL StartGame(BOOL bNewGame, BOOL bSinglePlayer)
 	return gbRunGameResult;
 }
 
-/**
- * @brief Save game configurations to ini file
- */
-static void SaveOptions()
+void SaveOptions()
```
Comment (AJenbo) [inline Source/diablo.cpp:393]:
```suggestion
/**
 * @brief Save game configurations to ini file
 */
static void SaveOptions()
```

File: Source/diablo.h [hunk 2]
```diff
@@ -143,6 +180,10 @@ extern int gnTickRate;
 extern WORD gnTickDelay;
 extern Options sgOptions;
 
+/**
+ * @brief Save game configurations to ini file
+ */
+void SaveOptions();
```
Comment (AJenbo) [inline Source/diablo.h:186]:
```suggestion

```

File: Source/diablo.h [hunk 3]
```diff
@@ -142,7 +179,6 @@ extern char sgbMouseDown;
 extern int gnTickRate;
 extern WORD gnTickDelay;
 extern Options sgOptions;
```
Comment (AJenbo) [inline Source/diablo.h:181]:
```suggestion
extern Options sgOptions;

```

**Pull request merged**